deployment/cdk/opensearch-service-migration/lib/common-utilities.ts

```diff
@@ -417,10 +417,10 @@ function parseAuth(json: any): ClusterAuth | null {
 
 export function parseClusterDefinition(json: any): ClusterYaml {
     const endpoint = json.endpoint
-    const version = json.version
     if (!endpoint) {
         throw new Error('Missing required field in cluster definition: endpoint')
     }
+    const version = json.version;
```

Comment:
can we do a check in stack-composer that this version exists on the source object since it's required so it's a synth failure?

Reply:
I did not throw an error on a missing value, it broke a sufficiently large number of test cases and I'd rather we reorganized how the validation of these parameters is handled that duplicate the logic in python typescript and java code in a future PR.